### DISCUSSION TITLE
Need syntax highlight support alone without other language features

### DISCUSSION THREAD

**Discussion by ManojkumarV** [microsoft/monaco-editor#3793] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

Since language features are provided by language server is it possible to disable additional language features provided for Typescript and Javascript by default. If possible, what are the steps to remove language features.

**Comment by hediet** [comment]
See https://github.com/microsoft/monaco-editor/issues/3126
